migrations/sql/V1.7__Add_Documents.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE public.document_manager (
+	document_manager_id serial NOT NULL,
+	document_guid uuid NOT NULL DEFAULT gen_random_uuid(),
+	full_storage_path varchar(100) NOT NULL,
+	upload_date timestamptz NOT NULL,
+	file_display_name varchar(40) NOT NULL,
+	path_display_name varchar(100) NOT NULL,
+	create_user varchar(60) NOT NULL,
+	create_timestamp timestamptz NOT NULL DEFAULT now(),
+	update_user varchar(60) NOT NULL,
+	update_timestamp timestamptz NOT NULL DEFAULT now(),
+	CONSTRAINT document_manager_pk PRIMARY KEY (document_manager_id)
+);
+
+-- Permissions
+
+ALTER TABLE public.document_manager OWNER TO mds;
```

Comment:
We have an aftermigrate script that grants all permissions to the mds user, so feel free to remove this statement :)